pages/index.html

```diff
@@ -1,12 +1,29 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="/styles/index_styles.css">
 </head>
+
 <body>
-  
+    <header>
+        <h1> Ashley Benson</h1>
+        <nav>
+            <a href="http://127.0.0.1:5500/pages/about.html" id="about">About</a>
+            <a href="http://127.0.0.1:5500/pages/index.html" id="home">Home</a>
+            <a href="http://127.0.0.1:5500/pages/portfolio.html" id="portfolio">Portfolio</a>
+
+        </nav>
+        <img src="/images/butterfly.jpeg" alt="header_img" id="header_img">
```

Comment:
For some reason, it feels odd to me to have this image (which mostly fills the page and feels more like the main content) here in the `header`.

I'd probably put it together with the following `p` tag in a `main` content tag.